src/2020-06-08-mixing-modules-and-export.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Mixing Rcpp modules and Rcpp attributes"
+author: "Bob Jansen"
+license: GPL (>= 2)
+tags: basics attributes modules
```

Comment:
We don't have a tag `attributes` and I have been reluctant to add new ones.  Just `basics modules` seems good.